tests/Dfe.PlanTech.Web.E2ETests/cypress/helpers/text-helpers.js

```diff
@@ -7,7 +7,8 @@ const ReplaceWhiteSpace = (string) =>
     .replace(/\s/g, " ")
     .replace(" ", " ")
     .replace(" ", " ")
```

Comment:
I've checked the char codes and lines 8 & 9 both appear to be spaces so I think we can delete them both

Reply:
A rogue source appeared and said these were meant to be U+00A0 and U+202F
I propose we change the code to this
`string.replace(/\s|\u00A0|\u202f|\u0020|\u2007/g, " ").replace(/\s+/g, " ")`
which I believe is the space like characters that could possibly appear

Reply:
Done this, needed one of the other .replaces but working now